src/main/java/uk/gov/hmcts/reform/orgrolemapping/domain/model/OrganisationInfo.java

```diff
@@ -16,6 +16,6 @@ public class OrganisationInfo {
 
     private String organisationIdentifier;
     private String status;
-    private LocalDateTime lastUpdated;
+    private LocalDateTime organisationLastUpdated;
```

Comment:
PRD will return field as `lastUpdated` - so you need to do 1 of the following

- [ ] revert this change & in the `OrganisationRefreshQueueRepository.java` do => `paramValues.addValue(ORGANISATION_LAST_UPDATED, r.getlastUpdated());` 
- [ ] annotate field with `@JsonProperty("lastUpdated")`

Reply:
I opted for the first but now I wish I went for the second option. The domain model should match the entity imo